src/richie/apps/courses/templates/courses/cms/fragment_course_glimpse.html

```diff
@@ -30,7 +30,7 @@
         </div>
         {% endif %}
         <div class="course-{{ course_variant }}__wrapper">
-            <p class="course-{{ course_variant }}__title">{{ course_page.get_title }}</p>
+            <h6 class="course-{{ course_variant }}__title" title="{{ main_organization_title }}">{{ course_page.get_title }}</h6>
```

Comment:
Why 6?
Worth a separate commit and mentioning in CHANGELOG?

Reply:
From a semantic point of view, I think more logical that course title is a title not a paragraph. WDYT?

If you are ok with this modification, I'll move it in a separate commit 👍

Reply:
Yes I agree. The level could also be passed by the parent like we do on some elements.